Cilsil.Test/Assets/Utils.cs

```diff
@@ -202,6 +224,21 @@ public static string GetString(InferError error)
         public static string Assign(VarName name, string value) =>
             $"{GetString(name)} = {value};\n";
 
+        /// <summary>
+        /// Generates a string representation of instantiating a new variable.
+        /// </summary>
+        /// <param name="type">The type of the variable being instantiated.</param>
+        /// <param name="name">The name of the variable being instantiated.</param>
+        /// <param name="value">The string representation of the value to be assigned.</param>
+        /// <param name="withLineEnding">True if that call should have a semicolon ending.</param>
+        /// <returns>String representation of the variable instantiation statement.</returns>
+        private static string Instantiate(VarType type,
```

Comment:
This we don't need -- in other tests, we simply do Declare into Assign as follows: https://github.com/xi-liu-ds/infersharp/blob/5b6caacd9122bf58f16e9b3b50340600b935a439/Cilsil.Test/Assets/Utils.cs#L278

Reply:
Done

Reply:
> This we don't need -- in other tests, we simply do Declare into Assign as follows: https://github.com/xi-liu-ds/infersharp/blob/5b6caacd9122bf58f16e9b3b50340600b935a439/Cilsil.Test/Assets/Utils.cs#L278

On second thought. In this way we plant a resource leak/null dereference forever in our code base, which is not good. Revert all the changes

Reply:
> This we don't need -- in other tests, we simply do Declare into Assign as follows: https://github.com/xi-liu-ds/infersharp/blob/5b6caacd9122bf58f16e9b3b50340600b935a439/Cilsil.Test/Assets/Utils.cs#L278

This one wouldn't work in our case because we want to declare before the try block and instantiate inside of try block